### DISCUSSION TITLE
Do not limit account changes to Mastodon

### DISCUSSION THREAD

**Discussion by linux-lukas** [mastodon/mastodon#16881] [General]
Have there been any thoughts about extending the account switching function to allow migration to other Fediverse services such as Friendica, Pixelfed, etc.?

**Comment by Gargron** [comment]
There is nothing Mastodon specific there, those services can support the flow if they wish

**Reply by linux-lukas** [reply]
Does a migration from e.g. Mastodon to Pixelfed already work now?

**Reply by trwnh** [reply]
> Does a migration from e.g. Mastodon to Pixelfed already work now?

Pixelfed would have to add support for Move activities and aliases first.
